pkg/graph/resolvers/system_intake_actions.go

```diff
@@ -590,6 +606,20 @@ func CreateSystemIntakeActionCloseRequest(
 			return nil, err
 		}
 	}
+	if emailClient != nil && input.NotificationRecipients != nil { // Don't email if no recipients are provided or there isn't an email client
+		err = emailClient.SystemIntake.SendCloseRequestNotification(ctx,
+			*input.NotificationRecipients,
+			intake.ID,
+			intake.ProjectName.ValueOrZero(),
+			intake.Requester,
+			*input.Reason,
```

Comment:
As with the reopen email, this is a possible nil-dereference error, due to the reason field being optional; treat it the same way as in the reopen email.